pkgs/tools/archivers/torrent7z/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, fetchpatch, lib, ncurses }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, ncurses }:
```
`fetchpatch` is not used

Arguments should be listed in the order used with the exception of lib, which always goes first